docs/concepts/why-filecoin.md

```diff
@@ -0,0 +1,79 @@
+
+## Reputation not marketing
+
+In Filecoin, storage providers prove their reliability through their track record published on the blockchain, not through marketing claims published by the provider themselves. Users don’t need to rely on status pages or self-reported statistics from storage providers.
+
+## Choice of tradeoffs
+
+Users get to choose their own tradeoffs between cost, redundancy and speed. Users are not limited to a set list of data centres offered by their provider, but can choose to store their files on any miner participating in Filecoin.
+
+## Puts empty storage to use
+
+Filecoin puts disk space that would otherwise sit empty to use. Miners are incentivised to join Filecoin because they get paid for their participation, unlike other distributed networks where there is no economic incentive. Filecoin also makes it easy to join because miners self-manage their storage, earning Filecoin tokens without needing human supervision.
+
+## Censorship resistance
+
+Filecoin resists censorship because there is no central provider that can be coerced into deleting files or withholding service. The network is made up of many different computers run by many different people and organizations. Faulty or malicious actors are noticed by the network and removed automatically.
+
+## Useful blockchain
+
+In Filecoin, miners are rewarded for providing storage, not for performing wasteful computations. Filecoin secures its blockchain using proof of file replication and proof of storage over time. It doesn’t rely on energy intensive proof-of-work schemes like other blockchains. Miners are incentivised to amass hard drives and put them to use by storing files. Filecoin doesn’t incentivise hoarding of graphics cards or application-specific integrated circuits for the sole purpose of mining.
+
+## Provides storage to other blockchains
+
+Filecoin excels at storing large files, whereas other blockchains can typically only store very small amounts of data, very expensively. Filecoin can provide its storage to other blockchains, allowing them to store large files. In the future, mechanisms will be added to Filecoin enabling transactions on Filecoin’s blockchain to interoperate with transactions on other blockchains.
+
+## Content addressing
+
+Files are referred to by the data they contain, not by fragile identifiers such as URLs. Files remain available no matter where they are hosted or who they are hosted by. When a file becomes popular it can be quickly distributed by swarms of computers instead of relying on a central computer which easily becomes overloaded by network traffic.
+
+When multiple users store the same file (and choose to make the file public by not encrypting it), everyone who wants to download the file benefits from Filecoin keeping it available. No matter where a file is downloaded from, downloaders can verify they have received the correct file and that it is intact.
```

Comment:
This paragraph makes it sound like it's easy to download a file that's been stored on Filecoin, which isn't the impression I have from discussions about the retrieval process. @mishmosh can say better than I can whether this is a fair representation. 
```suggestion
When multiple users store the same file (and choose to make the file public by not encrypting it), everyone who wants to download the file benefits from Filecoin keeping it available. No matter where a file is downloaded from, downloaders can verify that they have received the correct file and that it is intact.
```

Reply:
Good call, this was probably too generous a representation. I’ve removed the paragraph.